llvm/test/tools/sycl-post-link/spec-constants/multiple-composite-usages.ll

```diff
@@ -39,7 +39,7 @@ target triple = "spir64-unknown-unknown-sycldevice"
 
 $_ZTS4Test = comdat any
 
-@__builtin_unique_stable_name._ZNK2cl4sycl6ONEAPI12experimental13spec_constantI3PODS4_E3getIS4_EENSt9enable_ifIXsr3std6is_podIT_EE5valueES8_E4typeEv = private unnamed_addr addrspace(1) constant [9 x i8] c"_ZTS3POD\00", align 1
+@__builtin_unique_stable_name._ZNK2cl4sycl3ext6oneapi12experimental13spec_constantI3PODS4_E3getIS4_EENSt9enable_ifIXsr3std6is_podIT_EE5valueES8_E4typeEv = private unnamed_addr addrspace(1) constant [9 x i8] c"_ZTS3POD\00", align 1
```

Comment:
I also think that those changes are not strictly necessary and from test readability point of view I would prefer shorter made up names rather than ones which are strictly the same as generated by FE from Headers

Reply:
Reverted

Reply:
Note: there are 4 more spec constant tests + a bunch of ESIMD tests which also can be left as-is, I guess (but better ask @dendibakh or @kbobrovs)